tests/test_instant_command.py

```diff
@@ -1,14 +1,15 @@
 import commands2
-from util import ConditionHolder
+from CommandTestBase import ConditionHolderRequirement
+from commands2.cmd import run_once
 
 
 def test_instant_command(scheduler: commands2.CommandScheduler):
-    cond = ConditionHolder()
+    cond = ConditionHolderRequirement()
 
-    cmd = commands2.InstantCommand(cond.setTrue)
+    cmd = run_once(cond.set_true, {cond})
 
     scheduler.schedule(cmd)
     scheduler.run()
 
-    assert cond.getCondition()
-    assert not scheduler.isScheduled(cmd)
+    assert cond.get_condition()
+    assert not scheduler.is_scheduled(cmd)
```

Comment:
If we were to provide a python implementation of commands, a primary goal would be to attempt to keep backwards compat with the current commands so that teams don't have to rewrite everything next year. Historically, we've used java camelcase conventions for function names, so we should stick to that.

Ideally, all of this year's tests (and any tests written for this effort that aren't coroutine related) should pass without modification on either the C++ codebase or the pure python implementation.